fastlane/Snapfile

```diff
@@ -0,0 +1,13 @@
+devices([
+   "iPhone 8 Plus"
+])
+
+languages([
+  "en-US"
+])
+
+scheme("SimplenoteScreenshots")
+
+output_directory("./screenshots")
```

Comment:
This is your culprit for the directory thing – if you change this to `./fastlane/screenshots` it _should_ fix the issue, but that's off the top of my head 🙃